Sources/Views/EpisodeVideoView.swift

```diff
@@ -93,6 +100,10 @@ private func progressPollingScript(isEpisodeViewable: Bool) -> Node {
               httpRequest.send();
             }
           });
+          player.setPlaybackRate( localStorage.getItem('pf_playbackrate') || 1.0);
+          player.on('playbackratechange', function(data) {
+            localStorage.setItem('pf_playbackrate', data.playbackRate);
+          });      
```

Comment:
I belive the same change can be done here:

```suggestion
          const defaultPlaybackRate = 1.0;
          const playbackRate = parseFloat(localStorage.getItem('pf_playbackrate')) || defaultPlaybackRate;
          player.setPlaybackRate(playbackRate);

          player.on('playbackratechange', function(data) {
            try {
              localStorage.setItem('pf_playbackrate', String(data.playbackRate));
            } catch (e) {
              console.error('Failed to save playback rate:', e);
            }
          });
```